### ISSUE TITLE
Custom background images

### ISSUE THREAD

**Issue by l-Noviem-l** [Superkat32/YouWereSlain#7]
This is more of a suggestion than an issue, but maybe you can add some sort of config to add an image to the background?
I'm very unfamiliar with modding so I'm not sure if this can be added but it'd be nice **:)**

**Comment by Superkat32** [comment]
I think the general idea is a good one, but I sorta want to hear more. I specifically want to hear more on how you would want it to be implemented, and the use case of the suggestion(because I couldn't really think of any).

For example, I could have the implementation completely hide the world, with only text and buttons shown in front of the image. But, I could also have it so that the image's size and position can be configured. There could also be the ability to have multiple images, either chosen at random, or all shown at the same time.  

As for the use case, how do you plan on using it? The only use case I could really come up with myself is displaying memes on the death screen(which would require the adjustable size/position, and the ability to add multiple images.)

I'm happy to add it, I don't think it would be terribly difficult, but I'm curious how you want it to be done because I'm unsure how I want it to be done. (Also, if you didn't know, using a GitHub issue for a suggestion is quite common, and typically the only way to get a suggestion to a developer in most cases. If you did know, then sorry for making you read extra text, lol)

**Comment by l-Noviem-l** [comment]
Well the idea I had going for was the Pizza Tower "technical difficulties" screen. Like an background image of some sort. Maybe some sort of option for it to fade in as well? I suppose the "adding multiple images" like you said would be sick. Like I said I don't have any experience in modding, this is a very new thing for me lmao.

**Comment by Superkat32** [comment]
Ah, yeah(unfamiliar with Pizza Tower I had to look it up lol), I think that would be fun. I'll see what I can do, and try to send you some videos once I get some progress on it(which might be a few months, I'm not really sure). I'll try to have the implementation allow for both a full on background image replacement and memes, because I couldn't decide which one would be better.

**Comment by l-Noviem-l** [comment]
Yeah I was suspecting that you might have not heard of it. But sounds good!
I don't mind waiting.

**Comment by BioTechproject27** [comment]
I also came here to make that suggestion.
My use case(s) would be to make a dark red blur fade/creep in from the edges inwards.
Also memes (e.g. darksouls death screen).
